homework-submission/my-project/src/AddForm.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import DatePicker from 'react-datepicker';
+import 'react-datepicker/dist/react-datepicker.css';
+
+
+class AddTodo extends React.Component {
+    state = {
+        description: '',
+        deadline: '',
+        done: false,
+
+    }
+    newTodo = (e) => {
+        this.setState({
+            [e.target.id]: e.target.value
+        })
+        console.log(this.state)
+    }
+    newDate = (date) => {
+        this.setState({
+            deadline: date.format("YYYY-MM-DD"),
```

Comment:
you have two different types of data in your `state` one is here: {Date}, Other is in the initial state: {String}.
Make sure to have the same kind of types when ever change accords.